src/api/app/assets/javascripts/webui/application/package.js

```diff
@@ -145,3 +145,17 @@ function live_build_log_ready() {
     $('.start_refresh').click(start_refresh);
     $('.stop_refresh').click(stop_refresh);
 }
+
+$( document ).ready(function() {
+  $('.btn-more').click(function() {
+    var link = $(this);
+    $('.more_info').toggle(0, function() {
+      if (link.is(':visible')) {
```

Comment:
what about writing it like this:

`$(this).is(':visible') ? link.text('less info') : link.text('more info');`